x-pack/plugins/watcher/server/models/watch_status/watch_status.js

```diff
@@ -119,7 +119,7 @@ export class WatchStatus {
   }
 
   get lastFired() {
-    const actionStatus = max(this.actionStatuses, 'lastExecution');
+    const actionStatus = maxBy(this.actionStatuses, 'lastExecution');
```

Comment:
This appears to be an actual bug that the test caught. I don't see how this was working before based on the [lodash documentation](https://lodash.com/docs/4.17.15#max).

Reply:
Nice it got caught 👍